klass-dropwizard/klass-dropwizard-configuration-factories/klass-dropwizard-configuration-data-stores/klass-dropwizard-configuration-data-store-reladomo/src/test/resources/logback-test.xml

```diff
@@ -19,7 +19,7 @@
 <configuration packagingData="true">
     <appender name="Console" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%highlight(%-5level) %d{"HH:mm:ss", system} [%thread] \(%file:%line\) %logger{40} %msg%n</pattern>
+            <pattern>%highlight(%-5level) %cyan(%date{HH:mm:ss.SSS}) %gray(\(%file:%line\)) [%white(%thread)] %blue(%marker) {%magenta(%mdc)} %green(%logger): %message%n%rootException</pattern>
```

Comment:
Check whether we still need `system` in the pattern, for all `logback-test.xml` copies.